_episodes/11-connecting.md

```diff
@@ -0,0 +1,427 @@
+---
+title: "Connecting to a remote HPC system"
+teaching: 25
+exercises: 10
+questions:
+- "How do I log in to a remote HPC system?"
+objectives:
+- "Configure secure access to a remote HPC system."
+- "Connect to a remote HPC system."
+keypoints:
+- "An HPC system is a set of networked machines."
+- "HPC systems typically provide login nodes and a set of worker nodes."
+- "The resources found on independent (worker) nodes can vary in volume and
+  type (amount of RAM, processor architecture, availability of network mounted
+  filesystems, etc.)."
+- "Files saved on one node are available on all nodes."
+---
+
+## Secure Connections
+
+The first step in using a cluster is to establish a connection from our laptop
+to the cluster. When we are sitting at a computer (or standing, or holding it
+in our hands or on our wrists), we have come to expect a visual display with
+icons, widgets, and perhaps some windows or applications: a graphical user
+interface, or GUI. Since computer clusters are remote resources that we connect
+to over often slow or laggy interfaces (WiFi and VPNs especially), it is more
+practical to use a command-line interface, or CLI, in which commands and
+results are transmitted via text, only. Anything other than text (images, for
```

Comment:
```suggestion
icons, widgets, and perhaps some windows or applications: a _graphical user
interface_, or GUI. Since computer clusters are remote resources that we connect
to over often slow or laggy interfaces (WiFi and VPNs especially), it is more
practical to use a _command-line interface_, or CLI, in which commands and
results are transmitted via text, only. Anything other than text (images, for
```

add italics to emphasis the comparison you are making